org.eclipse.wildwebdeveloper.embedder.node/src/org/eclipse/wildwebdeveloper/embedder/node/InitializeLaunchConfigurations.java

```diff
@@ -14,21 +14,26 @@
  *   Pierre-Yves B. - Issue #238 Why does wildweb do "/bin/bash -c which node" ?
  *   Pierre-Yves B. - Issue #268 Incorrect default Node.js location for macOS
  *******************************************************************************/
-package org.eclipse.wildwebdeveloper;
+package org.eclipse.wildwebdeveloper.embedder.node;
```

Comment:
should we rename the class by the way, to something like `NodeJSManager` or `NodeJSAccessor` ?

Reply:
I like `NodeJSManager` or `NodeJSRuntimeProvider`. But the first name is better as it's shorter.